src/status_im/ui/screens/desktop/main/chat/views.cljs

```diff
@@ -180,37 +180,42 @@
                     :current-public-key current-public-key)]))]]
        [connectivity/error-view]])))
 
-(views/defview chat-text-input []
+(views/defview chat-text-input [{:keys [set-container-height-fn]}]
```

Comment:
Seems that we don't need `set-container-height-fn` as fn input param (it's defined as a let binding below)

Reply:
@siphiuel, eagle eye! Thanks for catching, fixed!